core/templates/mathjaxConfig.ts

```diff
@@ -1,4 +1,5 @@
 window.MathJax = {
+  // @ts-ignore
```

Comment:
Did this because technically the type error thrown here was correct. This is not the type of `MathJax` but `MathJaxConfig`. To not do this we would have to declare the custom mathjax config the other way suggested [here](https://docs.mathjax.org/en/v2.7-latest/configuration.html).

Reply:
Any specific reason for why can't we follow the custom declaration?

Reply:
Yup, MathJax is a `namespace` itself. Also if we make the type of window.MathJax equal to `MathJax.Config` it finally becomes `MathJax.Config & typeof MathJax`. And I think it makes sense because technically window.MathJax should be the MathJax namespace, it shouldn't be just the config, right?